cli/src/printer.rs

```diff
@@ -115,7 +115,7 @@ impl DisplayTable for Project {
 
 impl DisplayTable for Source {
     fn to_table_headers() -> Row {
-        row![bFg => "Name", "ID", "Updated (UTC)", "Title"]
+        row![bFg => "Name", "ID", "Updated (UTC)", "Title", "Bucket"]
```

Comment:
Shouldn't this be `"Transform Tag"`? Maybe a bucket field is needed too? May also be best to leave `Title` at the end since it tends to be long.

Reply:
Yep you're right